src/mcl.ts

```diff
@@ -128,6 +136,43 @@ export function sign(
     return { signature, M };
 }
 
+export function verify(
+    signature: solG1,
+    pubkey: solG2,
+    message: Message
+): boolean {
```

Comment:
I replicate the function signature of the Hubble contracts, see 

https://github.com/thehubbleproject/hubble-contracts/blob/830341ae153493b3b23038d772b2fe27e90d87d6/test/bls.test.ts#L79